tests/integration/pob_suite.go

```diff
@@ -35,24 +39,250 @@ func NewPOBIntegrationTestSuiteFromSpec(spec *interchaintest.ChainSpec) *POBInte
+	s.Require().Equal(buildertypes.DefaultReserveFee, params.ReserveFee)
+	s.Require().Equal(buildertypes.DefaultMinBidIncrement, params.MinBidIncrement)
+	s.Require().Equal(buildertypes.DefaultFrontRunningProtection, params.FrontRunningProtection)
+	s.Require().Equal(buildertypes.DefaultProposerFee, params.ProposerFee)
+}
+
+func (s *POBIntegrationTestSuite) TestValidBids() {
+	params := QueryBuilderParams(s.T(), s.chain)
+	escrowAddr := sdk.AccAddress(params.EscrowAccountAddress).String()
+
+	s.Run("Valid Auction Bid", func() {
+		// get escrow account balance before bid
+		escrowAcctBalanceBeforeBid := QueryAccountBalance(s.T(), s.chain, escrowAddr, params.ReserveFee.Denom)
+
+		// create bundle w/ a single tx
+		// create message send tx
+		tx := banktypes.NewMsgSend(s.user1.Address(), s.user2.Address(), sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(100))))
+
+		// create the MsgAuctioBid
+		bidAmt := params.ReserveFee
+		bid, bundledTxs := CreateAuctionBidMsg(s.T(), context.Background(), s.user1, s.chain.(*cosmos.CosmosChain), bidAmt, []Tx{
+			{
+				User: s.user1,
+				Msgs: []sdk.Msg{
+					tx,
+				},
+				SequenceIncrement: 1,
+			},
+		})
+
+		height, err := s.chain.(*cosmos.CosmosChain).Height(context.Background())
+		require.NoError(s.T(), err)
+
+		// broadcast + wait for the tx to be included in a block
+		res := BroadcastMsgWithBlock(s.T(), context.Background(), s.user1, s.chain.(*cosmos.CosmosChain), height+2, bid)
+		height = uint64(res.Height)
+
+		// wait for next height
+		WaitForHeight(s.T(), s.chain.(*cosmos.CosmosChain), height)
+
+		// query + verify the block
+		block := Block(s.T(), s.chain.(*cosmos.CosmosChain), int64(height))
+		VerifyBlock(s.T(), block, res.TxHash, bundledTxs)
+
+		// ensure that the escrow account has the correct balance
+		escrowAcctBalanceAfterBid := QueryAccountBalance(s.T(), s.chain, escrowAddr, params.ReserveFee.Denom)
+		expectedIncrement := escrowAddressIncrement(bidAmt.Amount, params.ProposerFee)
+		require.Equal(s.T(), escrowAcctBalanceBeforeBid+expectedIncrement, escrowAcctBalanceAfterBid)
+	})
+
+	s.Run("Valid bid with multiple other transactions", func() {
+		// get escrow account balance before bid
+		escrowAcctBalanceBeforeBid := QueryAccountBalance(s.T(), s.chain, escrowAddr, params.ReserveFee.Denom)
+
+		// create the bundle w/ a single tx
+		// bank-send msg
+		msgs := make([]sdk.Msg, 2)
+		msgs[0] = banktypes.NewMsgSend(s.user1.Address(), s.user2.Address(), sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(100))))
+		msgs[1] = banktypes.NewMsgSend(s.user2.Address(), s.user3.Address(), sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(100))))
+
+		// create the MsgAuctionBid
+		bidAmt := params.ReserveFee
+		bid, bundledTxs := CreateAuctionBidMsg(s.T(), context.Background(), s.user1, s.chain.(*cosmos.CosmosChain), bidAmt, []Tx{
+			{
+				User:              s.user1,
+				Msgs:              msgs[0:1],
+				SequenceIncrement: 1,
+			},
+		})
```

Comment:
this creates a single transaction with two messages. I would make this a separate test case. you need to create another `UserTx` here.

Reply:
wym?

Reply:
These test-cases are 1:1 w/ the current suite